openhands/core/config/agent_config.py

```diff
@@ -46,6 +46,8 @@ class AgentConfig(BaseModel):
     """Whether history should be truncated to continue the session when hitting LLM context length limit."""
     enable_som_visual_browsing: bool = Field(default=True)
     """Whether to enable SoM (Set of Marks) visual browsing."""
+    enable_plan_mode: bool = Field(default=False)
```

Comment:
Let's default to this for testing purpose

```suggestion
    enable_plan_mode: bool = Field(default=True)
```